src/Sylius/Bundle/CoreBundle/OAuth/UserProvider.php

```diff
@@ -95,6 +104,7 @@ public function __construct(
         $this->userFactory = $userFactory;
         $this->userManager = $userManager;
         $this->customerRepository = $customerRepository;
+        $this->eventDispatcher = $eventDispatcher;
```

Comment:
```suggestion
        if ($eventDispatcher === null) {
            @trigger_error(sprintf('Not passing a $eventDispatcher to %s constructor is deprecated since Sylius 1.7 and will be removed in Sylius 2.0.', self::class), \E_USER_DEPRECATED);
        }
        $this->eventDispatcher = $eventDispatcher;
```